lambdas/services/mock_data/pds_patient_9000000001_X4S4L_pcse.json

```diff
@@ -73,7 +73,6 @@
     "gender": "female",
     "birthDate": "2010-10-22",
     "multipleBirthInteger": 1,
-    "deceasedDateTime": "2010-10-22T00:00:00+00:00",
```

Comment:
Here I removed the deceased status from all our test patients, except 201 and 202 which are added to test ingesting deceased patients.